style.css

```diff
@@ -59,3 +64,327 @@
   margin: 0;
   padding: 0;
 }
+
+.main {
+  background: #f7f7f9;
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-content: center;
+}
```

Comment:
[OPTIONAL]
- [ ] Also you may consider adding some spacing (margin, padding) to your page, especially on the left side.
As you cam see how your design looks like :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198557616-c3ebb8f2-123c-4d26-8915-e5e76a409b11.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198557735-d7c4b185-03f2-4219-9457-d0f28f64ca60.png)